Library/.rubocop.yml

```diff
@@ -405,9 +405,16 @@ Style/NumericLiterals:
   Exclude:
     - "**/Brewfile"
 
-# OpenStruct is a nice helper.
+# TODO: These are pre-existing violations and should be fixed.
```

Comment:
How about adding `rubocop:todo` instead so more violations aren't added to these files?